From 1f07c886535b68b690cadb12f49b0e25cde47ed0 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 28 May 2014 16:36:24 +0800
Subject: [PATCH 11/11] ENGR00307764 wifi:fix first connection failure issue
 in wfd

Asynchronization of sink's getting IP and dnsmasq handshake would result
in first connection failure randomly in wifi display case. And property
to sync it will guarantee the sequence right.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 init/property_service.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/property_service.c b/init/property_service.c
index 3b206a2..ec6059b 100644
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -72,6 +72,7 @@ struct {
     { "gsm.",             AID_RADIO,    0 },
     { "persist.radio",    AID_RADIO,    0 },
     { "net.dns",          AID_RADIO,    0 },
+    { "net.dnsmasq.",     AID_NOBODY,   0 },
     { "sys.usb.config",   AID_RADIO,    0 },
     { "net.",             AID_SYSTEM,   0 },
     { "dev.",             AID_SYSTEM,   0 },
-- 
1.8.0

